readme.md

```diff
@@ -32,6 +32,20 @@ sortKeys({c: 0, a: 0, b: 0}, {
 //=> {c: 0, b: 0, a: 0}
 ```
 
+As of v5.x, the comparator function, if given, will be bound to the current object.  It means `this` is available to access the object's values, e.g.:
+
+```js
+sortkeys({{c: 0, a: 0, b: 1}}, {
+  // compare based on keys and values
+  compare: function(left, right) {
+      let lvalue = this[left];
+      let rvalue = this[right];
```

Comment:
Looking at the example here now, I'm realizing that it would be nicer to just passing in the object as the third parameter instead. That would make it work with arrow functions and be more explicit. I don't really like implicit this binding in general.